### ISSUE TITLE
What about Proguard?

### ISSUE THREAD

**Issue by pratikbutani** [dlazaro66/QRCodeReaderView#147]
What are the things that I have to write for proguard?

**Comment by glaucomp** [comment]
Any answer??

**Comment by pratikbutani** [comment]
Nope, I don't remember exactly but I think I've used another library. Its no longer maintained by the developer even not been updated in the last 5 years. You should try another library or you can fork and update as you want.

**Comment by glaucomp** [comment]
Appreciate mate 
I will do that
